samples/subsys/usb/mass/prj.conf

```diff
@@ -14,3 +14,4 @@ CONFIG_USB_MASS_STORAGE_LOG_LEVEL_ERR=y
 # If the target's code needs to do file operations, enable target's
 # FAT FS code. (Without this only the host can access the fs contents)
 #CONFIG_FILE_SYSTEM=y
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
Is CONFIG_MAIN_STACK_SIZE=1536 possible also sufficient?

Reply:
It is possible, minimal possible I suggest is 1088=1024+64.

Reply:
okay, let us set it to CONFIG_MAIN_STACK_SIZE=1536